readme.md

```diff
@@ -1,32 +1,138 @@
+  ![Repo size](https://img.shields.io/github/repo-size/rajivharlalka/Accio?color=white)
+  ![GitHub Maintained](https://img.shields.io/badge/Maintained%3F-yes-brightgreen.svg?v=103)
+  ![Open Source](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)
+
+</div>
+<!-- -------------------------------------------------------------------------------------------------------------------------------------------------- -->
+
+
+<p align="center">
+    <strong> Personal Blog Site Theme </strong>
+    <br />
+  <br />
+    <a href="https://accio-demo.netlify.app/"><strong>Accio Official Website »</strong></a>
+    <br />
+    <a href="https://accio-demo.netlify.app/blog/">View Demo</a>
+    ·
+    <a href="https://github.com/rajivharlalka/Accio/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rajivharlalka/Accio/issues">Request Feature</a>
+  </p>
+<!-- -------------------------------------------------------------------------------------------------------------------------------------------------- -->
+<!-- TABLE OF CONTENTS -->
+
+#### Table of Contents :
+* [About the Accio](#About-the-Accio)
+* [Open Source programs this repo has been part of](#Open-Source-programs-this-repo-has-been-part-of)
+* [Contribution Guidelines](#Contribution-Guidelines)
+* [Contributors](#Contributors)
+
+
+<br>
+<!-- -------------------------------------------------------------------------------------------------------------------------------------------------- -->
+
+<!-- ABOUT THE PROJECT -->
+
+## About the Accio
+
+<p align="center">
+ Accio is a Personal Blog Page using Zola Static Site Generator and a personal theme.
 Currently this project is actively in development mode.
+   
+</p>
+<p align="center">
+  <a href="https://accio-demo.netlify.app/">Accio</a> - checkout us.
+  <br>
+  <br>
+  
+   <img src="https://github.com/ayush-sleeping/Accio/blob/master/ss%20of%20Accio%20web.png" />
+</p>
+
 
-<br/>
+<br>
 
-### <div align="center">How to contribute ?</div>
+<br>
 
-1. Fork the repository.
-2. Setup your local machine. <a href="#setup">Read More</a>
-3. Make required changes.
-4. Create a pull request to the master branch.
-5. Get review from mentors and DONE!
+<br>
+<!-- -------------------------------------------------------------------------------------------------------------------------------------------------- -->
 
-<br/>
-<a href="https://discord.gg/6pGdW2rg">Discord Server Link</a>
-<br/><br/>
+## Open Source programs this repo has been part of
+<a href="https://github.com/Rayman-Sodhi/Clone-IT"><img src="https://badges.frapsoft.com/os/v1/open-source.svg?v=103"></a>
+
+<div align="center">
+<img src="https://media-exp1.licdn.com/dms/image/C560BAQGNflW7cZMukg/company-logo_200_200/0/1589034084843?e=1646870400&v=beta&t=mKip_R8iZ5ph54NuyudJ2dmcdp9hSuespgF4exp1ZQ0" width="150px">
+</div>
+
+<div align="center">
+    <a href="https://kwoc.kossiitkgp.org/">Kharagpur Winter of Code</a>
+
```

Comment:
Checkout this, Changed the required things ........

Reply:
Thanks for the quick changes, looks much better